NodeBase/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM selenium/base:2.52.0
+FROM selenium/base:latest
```

Comment:
I'm curious how this would work though, as the changes to node-firefox require the additional bzip package from base. Wouldn't using the version tag cause broken images to be built?